README.md

```diff
@@ -2,7 +2,7 @@ MonologBundle
 =============
 
 The `MonologBundle` provides integration of the [Monolog](https://github.com/Seldaek/monolog)
-library into the Symfony2 framework.
+library into the Symfony2/3/4 framework.
```

Comment:
What about using just Symfony (that's what we've been doing elsewhere)? `into the Symfony framework`

Reply:
that makes a lot of sense :+1: updated